test/parallel/test-tls-connect-no-host.js

```diff
@@ -8,10 +9,9 @@ const tls = require('tls');
 
 const assert = require('assert');
 const fs = require('fs');
-const path = require('path');
 
-const cert = fs.readFileSync(path.join(common.fixturesDir, 'test_cert.pem'));
-const key = fs.readFileSync(path.join(common.fixturesDir, 'test_key.pem'));
+const cert = fs.readFileSync(fixtures.path('test_cert.pem'));
+const key = fs.readFileSync(fixtures.path('test_key.pem'));
```

Comment:
Hello @dtex ! This is good but can be made even better using `fixtures.readSync`, e.g.

```js
const cert = fixtures.readSync('test_cert.pem');
const key = fixtures.readSync('test_key.pem');
```

Reply:
Thanks for pointing me to that. I've made that update.

I went ahead and amended the existing commit rather than add a second for such a small change. I hope that's okay.